src/dataframe/io.jl

```diff
@@ -801,7 +801,7 @@ function readtable(io::IO,
                    separator::Char = ',',
                    quotemark::Vector{Char} = ['"'],
                    decimal::Char = '.',
-                   nastrings::Vector = ["", "NA"],
+                   nastrings::Vector = ["", "NULL", "NA"],
```

Comment:
I agree with @quinnj here, I would much prefer if ``NULL`` values would not write out a ``NULL`` but just nothing, and if the readtable function wouldn't add ``NULL`` to the ``nastrings`` vector by default. These default ``nastrings`` are quite a problem already: I think they make sense for non-string columns, but having them there by default for string columns is not ideal at all, after all both "NA" or "NULL" are valid values for a string. And e.g. according to the iso alpha-2 code "NA" is the country code for Namibia, and there is a *lot* of data out there that uses that classification.